### PR TITLE
adds time ranges to log links in webapp

### PR DESCRIPTION
**Pull request by jonmcoe**
@josegonzalez

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,6 +1,12 @@
 Changelog
 =========
 
+
+0.12.1 (2016-08-15)
```
Comment (josegonzalez) [inline CHANGES.rst]: Please undo this change, it'll be automatically overwritten on release.

File: cronq/__init__.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-__version__ = '0.12.0'
+__version__ = '0.12.1'
```
Comment (josegonzalez) [inline cronq/__init__.py]: Same here.

File: README.rst [hunk 3]
```diff
@@ -143,7 +143,9 @@ The web view is a WSGI app run from ``cronq.web:app`` and requires only database
     #
     # {job_id} : replaced with the job's job_id
     # {run_id} : replaced with the job's run_id
-    export CRONQ_LOG_URL_TEMPLATE="https://logs.service/search?run_id={run_id}"
+    # {start_time} : replaced with job's start time
```
Comment (josegonzalez) [inline README.rst]: Can you add the time format that it will be set to?

**Comment by jonmcoe** [comment]
oops, hadnt noticed the "release" script. should be good now

**Comment by jonmcoe** [comment]
@josegonzalez bump

**Comment by josegonzalez** [comment]
Going to assume you tested this.

**Comment by jonmcoe** [comment]
yes ran the webapp only locally, pointed towards prod mysql
